recipes/scheil/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "scheil" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "de43e05fb9fdfb084fa3c7709000cc81c88292eaee24a0ac020c52c032c0c809" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3
+    - numpy
+    - scipy
+    - pycalphad >=0.8.1
+
+test:
+  imports:
+    - scheil
+    - scheil.simulate
+    - scheil.solidification_result
+    - scheil.utils
+
+about:
+  home: https://scheil.readthedocs.io
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A Scheil-Gulliver simulation tool using pycalphad'
+  doc_url: A Scheil-Gulliver simulation tool using pycalphad
+  dev_url: https://github.com/pycalphad/scheil
+
+extra:
+  recipe-maintainers:
+    - bocklund
+    - richardotis
```

Comment:
@richardotis are you OK with being a maintainer?

Reply:
Yes I consent to being a maintainer of this package